driver/driver_config_builder.py

```diff
@@ -272,7 +281,10 @@ def from_rds(self, db_instance_identifier) -> BaseDriverConfigBuilder:
             "db_host": get_db_hostname(db_instance_identifier, self.rds_client),
             "db_port": get_db_port(db_instance_identifier, self.rds_client),
             "db_version": get_db_version(db_instance_identifier, self.rds_client),
-            "db_type": get_db_type(db_instance_identifier, self.rds_client)
+            "db_type": get_db_type(db_instance_identifier, self.rds_client),
+            "db_non_default_parameters": get_db_non_default_parameters(
```

Comment:
A sample of resultant config, with the added part in bold:
`DriverConfig(server_url='https://dev.api.ottertune.com', db_identifier='bug-bash-pg-14', aws_region='us-east-2', db_type='postgres', db_host='bug-bash-pg-14.csmbpj48k8gh.us-east-2.rds.amazonaws.com', db_port=5432, db_version='14_2', db_user='ottertune', db_password='ottertune', db_name='postgres', api_key='cHNcTsM2qhxYgLc7KSXNee9DBQBPBdL+J1+pKmETcmk=', db_key='a468086d-192a-456d-96bd-14f23f138108', organization_id='978a5fa7-3a7f-4ad8-9a20-670d898736cf', monitor_interval=60, metric_source=['cloudwatch'], metrics_to_retrieve_from_source={'cloudwatch': ['BurstBalance', 'CPUUtilization', 'CPUCreditUsage', 'CPUCreditBalance', 'CPUSurplusCreditBalance', 'CPUSurplusCreditsCharged', 'DatabaseConnections', 'DiskQueueDepth', 'FreeableMemory', 'FreeStorageSpace', 'MaximumUsedTransactionIDs', 'NetworkReceiveThroughput', 'NetworkTransmitThroughput', 'OldestReplicationSlotLag', 'ReadIOPS', 'ReadLatency', 'ReadThroughput', 'ReplicaLag', 'ReplicationSlotDiskUsage', 'SwapUsage', 'TransactionLogsDiskUsage', 'TransactionLogsGeneration', 'WriteIOPS', 'WriteLatency', 'WriteThroughput']}, disable_table_level_stats=False, num_table_to_collect_stats=1000, table_level_monitor_interval=3600, disable_index_stats=False, num_index_to_collect_stats=10000, `**`db_non_default_parameters=['auto_explain.log_min_duration', 'autovacuum_analyze_scale_factor', 'autovacuum_vacuum_cost_delay', 'autovacuum_vacuum_cost_limit', 'autovacuum_vacuum_scale_factor', 'autovacuum_vacuum_threshold', 'bgwriter_delay', 'bgwriter_lru_maxpages', 'bgwriter_lru_multiplier', 'checkpoint_completion_target', 'default_statistics_target', 'effective_cache_size', 'effective_io_concurrency'])`**